app/views/hyrax/my/_admin_set_action_menu.html.erb

```diff
@@ -18,20 +18,37 @@
                     class: 'itemicon itemedit',
                     title: t("hyrax.dashboard.my.action.edit_admin_set") do %>
           <%= t("hyrax.dashboard.my.action.edit_admin_set") %>
+        <% end %>         
+      </li>
+    <% else %>
+      <li role="menuitem" tabindex="-1">
+        <%= link_to "#",
```

Comment:
I tried to find a reference that explained what `<%= link_to "#"` is doing.  Can you point me to a reference or some other documentation.